src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/logs/logs.component.html

```diff
@@ -19,10 +19,10 @@
                                 [textItem]="logText"
                                 fileName="cluster_log">
             </cd-download-button>
-            <button type="button"
-                    class="btn btn-light"
-                    [cdCopy2ClipboardButton]="logText"
-                    [byId]="false"></button>
+            <cd-copy-2-clipboard-button
```

Comment:
Does not work the first time, only when doing it a second time.
![Peek 2020-12-14 14-42](https://user-images.githubusercontent.com/1897962/102088025-9da77000-3e1a-11eb-946e-d1fbc912e060.gif)

Reply:
@votdev I checked this in master and its the same behaviour over there. So created a tracker for this https://tracker.ceph.com/issues/48601. Maybe we can deal with this in a different PR since its a different issue?

Reply:
Good catch @votdev ; @nizamial09 just confirmed what you said; yeah, it doesn't look like its directly related to the button itself, but how the page handles the stored text.